pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```
@@ -89,7 +89,7 @@ public static Object jsonPath(Object object, String jsonPath) {
     if (object instanceof String) {
       return PARSE_CONTEXT.parse((String) object).read(jsonPath, NO_PREDICATES);
     }
-    return PARSE_CONTEXT.parse(object).read(jsonPath, NO_PREDICATES);
+    return object == null ? null : PARSE_CONTEXT.parse(object).read(jsonPath, NO_PREDICATES);
```

Input `object` should not be `null`. IMO We can let it throw NPE